products/ipad.md

```diff
@@ -77,20 +98,41 @@ releases:
     eol: false
     link: https://support.apple.com/kb/SP787
 
+-   releaseCycle: "pro-3"
+    releaseLabel: "iPad Pro (3rd generation)"
+    releaseDate: 2018-11-07
+    discontinued: 2020-03-18
+    eol: false
+    link: https://support.apple.com/kb/SP785
+
 -   releaseCycle: "6"
     releaseLabel: "iPad (6th generation)"
     releaseDate: 2018-03-27
     discontinued: 2019-09-10
     eol: false
     link: https://support.apple.com/kb/SP774
 
+-   releaseCycle: "pro-2"
+    releaseLabel: "iPad Pro (2nd generation)"
+    releaseDate: 2017-06-05
```

Comment:
```suggestion
    releaseDate: 2017-06-13
```

Since we're using the Release Date elsewhere?

Reply:
My mistake, good catch !